341-350.ipynb

```diff
@@ -0,0 +1,301 @@
+    "        return \"s=0\"  \n",
+    "    return sum(input_list[max_index + 1:])\n",
+    "input_list = [3, 4, 10, 15, 2, 25, 102, 12, 15]\n",
+    "prob_345(input_list)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 84,
+   "id": "615d4bf0",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "3000"
+      ]
+     },
+     "execution_count": 84,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def prob_346(input_list):\n",
+    "    min_value = 0  \n",
+    "    last_min_index = -1  \n",
+    "    mul = 1  \n",
+    "    for index, num in enumerate(input_list):\n",
+    "        if num < min_value:\n",
+    "            min_value = num\n",
+    "            last_min_index = index\n",
+    "    if last_min_index == -1:\n",
+    "        return \"s=0\"\n",
+    "    for num in input_list[:last_min_index]:\n",
+    "        if num > 0:\n",
+    "            mul *= num\n",
+    "    return mul\n",
+    "input_list = [3, 4, 10, -15, -2, 25, -102, 12, 5]\n",
+    "prob_346(input_list)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 110,
+   "id": "ca05891b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(0, -4)"
+      ]
+     },
+     "execution_count": 110,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def prob_347(input_list, b):\n",
+    "    index_mul = 1\n",
+    "    num_sum = 0\n",
+    "    for index, num in enumerate(input_list):\n",
+    "        if num < b:\n",
+    "            index_mul *= index\n",
+    "            num_sum += num\n",
+    "    return index_mul, num_sum\n",
+    "input_list = [3, 4, 10, -5, -2, 25, 102, 12, 5]\n",
+    "prob_347(input_list, 4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 117,
+   "id": "12b5cc3b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "13"
+      ]
+     },
+     "execution_count": 117,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def prob_348(input_list):\n",
+    "    ind_sum = 0\n",
+    "    for index, num in enumerate(input_list):\n",
+    "        if isinstance(num, int) and num % 2 == 0:\n",
+    "            ind_sum += index\n",
+    "    return ind_sum\n",
+    "input_list = [3, 0.863, 10, -5, -2, 25, 10.2, 12, 5]\n",
+    "prob_348(input_list)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 120,
+   "id": "0354a41c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4.0"
+      ]
+     },
+     "execution_count": 120,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def prob_349(input_list):\n",
+    "    odd_indices = []\n",
+    "    for index, num in enumerate(input_list):\n",
+    "        if isinstance(num, int) and num % 2 == 1:\n",
```

Comment:
Can you please check the problem once again ? You need to convert items to integer.